src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -141,4 +183,48 @@ struct Sqrt : db::ComputeTag {
       sqrt_magnitude;
   using argument_tags = tmpl::list<Tag>;
 };
+
+/// @{
+/// \ingroup DataBoxTagsGroup
+/// \ingroup DataStructuresGroup
+/// The point-wise L2 norm of a tensor
+///
+/// \snippet Test_Magnitude.cpp pointwisel2norm_name
+template <typename Tag>
+struct PointwiseL2Norm : db::SimpleTag {
```

Comment:
How about you move these tags so that you can share the documentation between the functions and the tags

Reply:
done